package.json

```
@@ -31,5 +31,9 @@
     "prettier-plugin-tailwindcss": "^0.6.6",
     "tailwindcss": "^3.4.1",
     "typescript": "^5"
+  },
+  "lint-staged": {
+    "*.{vue, ts, js}": "eslint --fix",
+    "*.{js,jsx,ts,tsx,vue,css,html,md,json,cjs}": "prettier --write"
```

```suggestion
    "*": "prettier --write"
```

Prettierの設定 #68  と統一して、すべてのファイルにフォーマットを効かせる方向で良さそうかな？と思いました！
ご検討いただけると幸いです🙇

---

PrettierとESLintは別の機能なので、どちらも実行したい場合はどちらも記述する必要があります！👍

また、Prettier公式のドキュメントを確認したところ、以下のような設定例が載ってました
https://prettier.io/docs/en/install#git-hooks

```
"**/*": "prettier --write --ignore-unknown"
```

調べてみたところ、自分が提案した`"*": "prettier --write"`は不十分で、`"*"`として指定するとメインディレクトリのみに適用されて、サブディレクトリには適用されないみたいです……！
また、`--ignore-unknown`オプションをつけると、Prettierがサポートしてない拡張子のファイルは、監視？をスキップしてくれるみたいなので、こちらを記述するとパフォーマンスが向上するかと思います！

雑ですが、調査報告をDiscussionsに残してありますー！ https://github.com/lgtm-factory/lgtm-factory/discussions/57#discussioncomment-10421966